extensions/hibernate-validator/deployment/src/test/java/io/quarkus/hibernate/validator/test/CustomConfigurationViaBeansTest.java

```diff
@@ -55,6 +57,9 @@ public void testCustomConfigurationViaBeans() {
         assertThat(hibernateValidatorFactory.getScriptEvaluatorFactory()).isInstanceOf(MyScriptEvaluatorFactory.class);
         assertThat(hibernateValidatorFactory.getGetterPropertySelectionStrategy())
                 .isInstanceOf(MyGetterPropertySelectionStrategy.class);
+        // Waiting for https://hibernate.atlassian.net/browse/HV-1841 to be released
+        //assertThat(hibernateValidatorFactory.getPropertyNodeNameProvider())
+        //        .isInstanceOf(MyPropertyNodeNameProvider.class);
```

Comment:
If you want this enabled now, you can use AssertJ's `extracting` method to assert on the field

Reply:
Yeah, I just don't want to depend on some internal state that I might change. It can wait for the next HV release.